daal4py/sklearn/decomposition/_pca.py

```diff
@@ -139,7 +139,10 @@ def _fit_full_daal4py(self, X, n_components):
         else:
             self.noise_variance_ = 0.
 
-        self.n_samples_, self.n_features_ = n_samples, n_features
+        if sklearn_check_version('1.2'):
```

Comment:
what would happen in 1.3? or we expect fixed in other places by that time?

Reply:
`self.n_features_in_` is used everywhere in default PCA starting this (1.2) release.
`sklearn_check_version` is `>=`

Reply:
ok. thanks!